content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -85,7 +85,7 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    chmod +x kubectl
    mkdir -p ~/.local/bin/kubectl
    mv ./kubectl ~/.local/bin/kubectl
-   # and then add ~/.local/bin/kubectl to $PATH
+   # and then add ~/.local/bin to $PATH
```

Comment:
```suggestion
   # and then append (or prepend) ~/.local/bin to $PATH
```

Per suggestion from @sftim , I feel this sounds much better & technically correct than adding since we either append/prepend to the $PATH variable instead of adding to it. WDYT?

Reply:
I agree `append/prepend` sounds more technically appropriate and in is line with @sftim 's suggestion. Thank you for pointing this out and I will incorporate the needful changes.  Thanks.